### PR TITLE
Array and object iteratee with single-parameter functions

### PR DESCRIPTION
**Pull request by timendez**

### CODE REVIEW

File: data/functions.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-functions = {
+const functions = {
   array: [
     {func: _.chunk, name: '_.chunk'},
     {func: _.compact, name: '_.compact'},
     {func: _.concat, name: '_.concat'},
     {func: _.difference, name: '_.difference'},
-    {func: _.differenceBy, name: '_.differenceBy'},
+    {func: _.differenceBy, name: '_.differenceBy', type: 'iteratee'},
```
Comment (rhenetz) [inline data/functions.js]: How about we add a constant for this stringly typed `iteratee` value we're using so much. That's a tough word to see typos in...way too many vowels.

File: data/functions.js [hunk 2]
```diff
@@ -23,77 +23,77 @@ functions = {
...
+    {func: _.intersectionBy, name: '_.intersectionBy', type: 'iteratee'}, //TODO
     {func: _.intersectionWith, name: '_.intersectionWith'}, //TODO
     {func: _.join, name: '_.join'},
     {func: _.last, name: '_.last'},
     {func: _.lastIndexOf, name: '_.lastIndexOf'},
     {func: _.nth, name: '_.nth'},
     {func: _.pull, name: '_.pull', mutates: true},
     {func: _.pullAll, name: '_.pullAll', mutates: true},
-    {func: _.pullAllBy, name: '_.pullAllBy', mutates: true}, //TODO Support iteratee functions
+    {func: _.pullAllBy, name: '_.pullAllBy', mutates: true, type: 'iteratee'}, //TODO Support iteratee functions, multiple arguments
```
Comment (rhenetz) [inline data/functions.js]: Is the `mutates` key being used anywhere? I can't seem to find any usage.

**Review by rhenetz — APPROVED** [review]
Left a couple small suggestions and I see you already updated the `type:` key to be `inputType:` so you should be good to go once you take a look at my comments.

File: data/functions.js (hunk 1 above)
Comment (timendez) [reply data/functions.js]: ahaha yes i've caught myself plenty of times with incorrect vowels

**Pull request merged**